examples/website/src/shared/molecules/commentTemplate.tsx

```diff
@@ -214,7 +214,11 @@ export function ComposeComment(props: ComposeCommentProps) {
     },
     onError: error => {
       console.error('Error creating a post', error)
-      setcommentStateInfo("Something went wrong, your comment couldn't be saved")
+      if(error.message === "Comment Length should be maximum of 1000 characters"){
```

Comment:
Is there a more stable way of checking this than comparing strings? Can you setCommentStateInfo to be error.message?